### PR TITLE
OCP-590: allowing deploying to only none-primary shards

### PR DESCRIPTION
**Pull request by pawelziebaopti**
This change is [Reviewable](https://reviewable.io/reviews/ZaiusInc/app-sdk/116)

### CODE REVIEW

File: src/app/validation/tests/validateMeta.test.ts [hunk 1]
```diff
@@ -157,12 +158,12 @@ describe('validateMeta', () => {
       ['Invalid app.yml: meta.availability should only contain valid availability zones (us,bar) found: fuz']);
   });
 
-  it('detects if "us" shard is not included', async () => {
-    const manifest = {...appManifest, meta: {...appManifest.meta, availability: ['foo', 'bar']}};
+  it('allows only none-primary shards', async () => {
```
Comment (SoAG) [inline src/app/validation/tests/validateMeta.test.ts:161]:
```suggestion
  it('allows only non-primary shards', async () => {
```

**Review by SoAG — APPROVED** [review]

**Comment by SoAG** [comment]
Should this be merged into master branch?

**Comment by pawelziebaopti** [comment]
> Should this be merged into master branch?

there's a separate PR for master

**Comment by pawelziebaopti** [comment]
closing as the main change is already merged. I added a small styling change to 1.2.x branch directly

**Pull request closed without merging**